### ISSUE TITLE
"Complete CATcher smoke test ASAP" when already completed

### ISSUE THREAD

**Issue by alcen** [nus-cs2103-AY1920S2/forum#97]
Hi, could I stop receiving these emails? I have already completed CATcher last week.

I deleted the "alpha" repository. Maybe that is the problem?

**Comment by okkhoy** [comment]
Yes; the script could not check the alpha repository. Hence, you received the mail.

**Comment by alcen** [comment]
Should I restore the repository? It seemed like something that was only used for the testing, so I deleted it
